1.11/administering-clusters/licenses/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle:  Licenses
+title: Licenses
+menuWeight: 1100
+enterprise: true
+---
+
+Mesosphere uses licenses to ensure that customers respect their contract terms. The DC/OS licensing component tracks the state of a cluster license, collects information to check if any licensing terms have been breached, and supports operations for updating the license when a contract is extended or changed.
+
+A cluster license file contains all of the information to determine if the terms of a contract agreement has been breached. It also contains the set of keys necessary to encrypt and decrypt audit data collected by the DC/OS licensing component. The license text contains:
+
+- A globally unique [customer key](/1.11/installing/ent/custom/configuration/configuration-parameters/#customer-key-enterprise).
+- A private key used by the DC/OS licensing component to sign audit data.
+- A public key used to decrypt the audit data.
```

Comment:
` A public key used to decrypt the audit data checksum`

Reply:
verified on staging